collections/_posts/2023-03-13-Fourier_Conv.md

```diff
@@ -0,0 +1,108 @@
+
+&nbsp;
+
+The full principle of the spectral transforms is summarized in the following figure : 
+<div style="text-align:center">
+<img src="/collections/images/fourier_conv/spectral_transform.jpg" height=600></div>
+<p style="text-align: center;font-style:italic">Figure 3. Illustration of the spectral transform performed in a FFC Block.</p>
+
+&nbsp;
+
+**_NOTE:_**  Local Fourier Unit is adopted here by the authors as it slightly increases thhe performances when used in conjunction with the Fourier Unit. However, the main idea carried by the paper lies in the Fourier Unit and its global receptive field. In several other applications where the Fast Fourier Convolution is reemployed, the Local Fourier Unit is not retained.
+
+&nbsp;
+
+# Results
+
+The efficiency of their convolution is assessed on some classic computer vision tasks. In those tests, the authors have replaced the standard convolutions in several backbones by their hybrid convolutional blocks. This change is done with a quite low computational cost, as it increases the number of parameters and FLOPs by only 2 to 5%.
+
+The two figures below show the performances that they achieved on the ImageNet classification task and the Human Keypoint Detection on COCO.
+
+<div style="text-align:center">
+<img src="/collections/images/fourier_conv/results_imagenet.jpg" height=400></div>
+<p style="text-align: center;font-style:italic">Figure 4. Experiments results of plugging FCC into some networks on ImageNet.</p>
+
+<div style="text-align:center">
+<img src="/collections/images/fourier_conv/results_coco.jpg" height=400></div>
+<p style="text-align: center;font-style:italic">Figure 5. Performances achieved on the COCO val2017 dataset for human keypoint detection.</p>
+
+Additional results presented in the article include more performances assessment and an ablation study to demonstrate the relevance of the Local Fourier Unit and the cross-scale information fusion, and to choose the hyperparameters of the FFC (feature split ratio). 
+
+&nbsp;
+
+# Conclusion
+
+Fast Fourier Convolution is an alternative to standard spatial convolution that combines information at a local and global scale by taking advantage of Fourier transform to have a global receptive field. Replacing usual convolution with this hydrib one seems to be able to improve performances of CNNs at a small computational cost. It has been recently used in applications that benefit from global receptive fields, such as large mask inpainting[^4].
+
+# References
+[^1]: See post on : [Deformable Convolutional Networks](http://127.0.0.1:4000/2022/11/14/DeformableConvolutionalNetworks.html)
```

Comment:
The URL should point to the online website rather than the localhost :wink: 
```suggestion
[^1]: See post on : [Deformable Convolutional Networks](https://creatis-myriad.github.io/2022/11/14/DeformableConvolutionalNetworks.html)
```